assert/assertions.go

```diff
@@ -893,19 +893,25 @@ func containsElement(list interface{}, element interface{}) (ok, found bool) {
 		}
 	}()
 
-	if listKind == reflect.String {
+	switch listKind {
+	case reflect.String:
 		elementValue := reflect.ValueOf(element)
 		return true, strings.Contains(listValue.String(), elementValue.String())
-	}
-
-	if listKind == reflect.Map {
+	case reflect.Map:
 		mapKeys := listValue.MapKeys()
 		for i := 0; i < len(mapKeys); i++ {
 			if ObjectsAreEqual(mapKeys[i].Interface(), element) {
 				return true, true
 			}
 		}
 		return true, false
+	case reflect.Slice:
+		switch element := element.(type) {
+		case []byte:
+			if list, ok := list.([]byte); ok {
```

Comment:
should we have `return false, false` if `!ok`?

Reply:
I don't think so, at least with how the code is written - iIf list is a [][]byte, we want to fall through to the loop check on  line 917.
Maybe it's valid to say that if list is neither a []byte or a [][]byte, we should return `false, false`, but I haven't thought very hard about that case yet.